### ISSUE TITLE
Filtered-listings / Pre-select filters - additional enhancement

### ISSUE THREAD

**Issue by cigar69** [marcom-unimelb/unimelb-design-system#295]
RE: Pre-select filters

Currently the pre-filter option is available for `course types' ( Check boxes) on the filtered-listings page. 

We would like to see an additional enhancement, whereby the `discipline' (drop-down )  can also be pre-filtered.

**Comment by ptagell** [comment]
Could this potentially be confusing? The exposed filters are more obvious
when they are pre-filtered, but the drop down is far less obvious.

**Comment by cigar69** [comment]
If the user is coming from the Melbourne Medical School website study page and is then directed to a dedicated Bespoke one stop shop  Medicine, Dentistry and Health Sciences degree page, that by pre-filtering to the discipline of 'medicine' , where by the course type can also be narrowed further by filtering, would provide a better experience for the user, given the school website they have come from.

 Same for a user coming from School of Dental wishing to study degrees in dentistry.

Sent from my iPhone

On 30 Jul 2015, at 4:01 pm, Paul Tagell <notifications@github.com<mailto:notifications@github.com>> wrote:

Could this potentially be confusing? The exposed filters are more obvious
when they are pre-filtered, but the drop down is far less obvious.

**Comment by cigar69** [comment]
Hi Jason,
any movement on the Discipline filter function?

**Comment by cigar69** [comment]
Hi Jason,
any movement on the Discipline filter function?

**Comment by cigar69** [comment]
Hi Jason,
any movement on the Discipline filter function?

**Comment by waitingallday** [comment]
Hi Brendan, sorry for dropping this - the functionality was added but I need to check it against bespoke. Will get back to you today.

**Comment by cigar69** [comment]
No problem Jason

**Comment by waitingallday** [comment]
Hi Brendan, I added this in a while back and completely forgot about it (even documented it on web.unimelb!), basically it will pre filter to the section as you requested by adding a `?section=` to the url. eg.

http://mdhs-study.unimelb.edu.au/degrees?section=Dentistry

I noticed a bug in the way this is implemented, it won't match any category title with a space which, unfortunately, is how bespoke is rendering them. I've pushed up a fix for this, we'll test it and deploy first thing monday.
